examples/echo_websocket_org.py

```diff
@@ -0,0 +1,9 @@
+import uwebsockets.client
+
+websocket = uwebsockets.client.connect("ws://echo.websocket.org/")
+
+websocket.send("fo0bAr\r\n")
```

Comment:
I dislike l33t speak :-P

Reply:
Done (assert and no l33t): https://github.com/Hermann-SW/uwebsockets/commit/2152a685371adf561448f023b091c85859cfd77e
```
>>> import echo_websocket_org
The quick brown fox jumps over the lazy dog

>>> 
```